rbc/heavydb/mathimpl.py

```diff
@@ -75,22 +83,31 @@ class Math_converter(ConcreteTemplate):
 binarys = []
 binarys += [("copysign", "copysignf", math.copysign)]
 binarys += [("atan2", "atan2f", math.atan2)]
-binarys += [("pow", "powf", math.pow)]
 binarys += [("fmod", "fmodf", math.fmod)]
 binarys += [("hypot", "hypotf", math.hypot)]
 binarys += [("remainder", "remainderf", math.remainder)]
 
 
 def impl_unary(fname, key, typ):
-    cpu = gen_codegen(fname)
+    if fname in INTR_TO_CMATH.values():
+        # use llvm intrinsics when possible
+        cpu = gen_codegen(f'llvm.{fname}')
+    else:
+        cpu = gen_codegen(fname)
```

Comment:
There is no `llvm.truncf` intrinsic. Suggest:
```suggestion
    for llvm_intr, intr in INTR_TO_CMATH.items():
        if fname == intr:
            cpu = gen_codegen(llvm_intr)
            break
    else:
        cpu = gen_codegen(fname)
```